src/styles/base/_buttons.scss

```diff
@@ -0,0 +1,235 @@
+.white-button {
+  width: 258px;
+  height: 60px;
+  border-radius: 10px;
+  background-color: #ffffff;
+  border: none;
+
+  &:hover {
+    background: none;
+    border: 1px solid white;
+
+    .button-text {
+      color: white;
+    }
+  }
+
+}
+
+.button-text {
+  position: relative;
+
+  width: 175px;
+  height: 24px;
+  font-family: CenturyGothic;
+  font-size: 16px;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: 1.5;
+  letter-spacing: 0.2px;
+  color: #1a3cfa;
+
+  &:after {
+    content: '\e901';
+    position: absolute;
+    font-family: icomoon;
+    font-size: 20px;
+    right: -25px;
+    top: 50%;
+    transform: translateY(-50%)
+  }
+}
+
+.clear-button {
+  width: 120px;
+  height: 50px;
+  background: none;
+  border-radius: 10px;
+  border: solid 1px #ffffff;
+  color: #ffffff;
+
+  &:hover {
+    background-color: white;
+
+    .clear-button-text {
+      color: #1a3cfa;
+    }
+
+    // color: 
+  }
+}
+
+.clear-button-text {
+  position: relative;
+  height: 24px;
+  padding-left: 10px;
+  font-size: 14px;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: 1.71;
+  letter-spacing: 0.2px;
+  text-align: right;
+
+  // color: white;
+  &:before {
+    content: '\e902';
+    position: absolute;
+    font-family: icomoon;
+    font-size: 14px;
+    left: -20px;
+    top: 50%;
+    transform: translateY(-50%);
+  }
+}
+
+.button-learn-more {
+  background-color: white;
+  ;
+  width: 160px;
+  height: 60px;
+  border-radius: 10px;
+  border: solid 1px #1a3cfa;
+
+  &:hover {
+    background: #1a3cfa;
+
+    .button-learn-more-text {
+      color: white;
+    }
+  }
+}
+
+.button-learn-more-text {
+  font-family: CenturyGothic;
+  position: relative;
+  font-size: 16px;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: 1.5;
+  letter-spacing: 0.2px;
+  text-align: center;
+  color: #1a3cfa;
+
+
+
+  &:after {
+    font-family: icomoon;
+    content: '\e901';
+    position: absolute;
+    right: -20px;
+    top: 50%;
+    transform: translateY(-50%);
+  }
+}
+
+.button-download-report {
+  width: 281px;
+  height: 60px;
+  // opacity: 0.3;
+  border-radius: 10px;
+  border: solid 1px #ffffff;
+  background: none;
+
+  &:hover {
+    background-color: white;
+
+    .text-button-download-report {
+      color: #1a3cfa;
+    }
+  }
+}
+
+.text-button-download-report {
+  font-family: CenturyGothic;
```

Comment:
Create global styles for buttons. Generic class `.btn` will contain most global styles and you won't repeat the same styles again and again.